omod/src/main/resources/webModuleApplicationContext.xml

```diff
@@ -129,11 +139,39 @@
 				
 				<prop key="admin/provider/providerAttributeType.list">providerAttributeTypeList</prop>
 				<prop key="admin/provider/providerAttributeType.form">providerAttributeTypeForm</prop>	
-				
+
  			</props>
 		</property>
 	</bean>	
 
+ 	<!-- index pages - begin -->  
+	
+	<!-- admin -->
+	<mvc:view-controller view-name="module/legacyui/admin/index" path="admin/index.htm"/>				
```

Comment:
@kristopherschmidt can you refer to my earlier comment as to what I thought was different about /admin and @wluyima 's response indicating that the problem couldn't be avoided by even using the servet api. (That was my understanding of his response anyway.)
Btw I haven't done any java in over 10 years, I have been mainly a .NET developer so this stuff is fairly new to me.